collections/guides/quickstarts/quickstart_nodejs.md

```diff
@@ -0,0 +1,244 @@
+
+If you are already familiar with Basis Theory and our platform in general, our guide on [How to send Token data via HTTP](https://guides.basistheory.com/guides/use-token-data-in-http-requests/) or [Collecting payments with Elements](https://guides.basistheory.com/guides/collect-atomic-cards-with-elements/) may be a better place to start.
+
+## Step 1: Get your Node.js environment ready
+
+### Step 1.1: Ensure you have Node.js installed
+
+Completing this tutorial will require you to have Node.js installed on your local system or have an environment you can run your script in. If you’re looking for help getting Node.js installed for your system — check out their [guide on installing](https://nodejs.org/en/download/package-manager/).
+
+### Step 1.2: Create a Node.js script
+To start, you’ll need a new Node.js file. Use the following commands (or however you create files) to create a file called `basistheory.js`:
+
+```bash
+//Mac or Linux
+touch basistheory.js
+
+//Windows
+type nul > basistheory.js
+```
+
+### Step 2: Import needed dependencies
+
+### Step 2.1: Install the `axios` npm package
+
+To simplify our request to Basis Theory, we will install a helper package called `axios`. This package simplifies HTTP requests in Node.js.
+```bash
+    npm install axios
+```
+
+### Step 2.2 Import the `axios` dependency in your script
+
+The simplest form of securing data with Basis Theory is through our API with a JSON body. In Node.js, this requires a few different libraries to make an API call. At the top of your file, you’ll import the following dependencies:
+
+```js
+const axios = require('axios');
+```
+
+## Step 3: Create a new Application in your Basis Theory Account
+
+If you don’t already have a Basis Theory account and your first Tenant, create one [here](https://portal.basistheory.com/register).
+
+Within your Basis Theory account, create a new Application. To create a new application, head to our portal [here](https://portal.basistheory.com/applications/create) — this Application’s API Key will enable you to authenticate with the Basis Theory platform and create Tokens within your Tenant.
+
+### 3.1 Enter an Application Name
+This name allows you to identify this application in the future — for this tutorial enter “Ruby Quickstart”.
+
+<img src="/assets/images/getting_started/application_name_nodejs.png" />
+
+### 3.2 Select Server-to-Server Application Type
+The Server-to-Server Application Type enables server-side applications to integrate with the Basis theory Platforms directly.
+
+<img src="/assets/images/getting_started/application_type.png" />
+
+### 3.3 Select Permissions
+Select `token:general:create` and `token:general:read` [permissions](https://docs.basistheory.com/api-reference/#permissions-permission-object) with the default “Low Impact”. These two permissions allow your Application to create a new Token and read the value back when you need to access it.
```

Comment:
```suggestion
Select `token:general:create` and `token:general:read` [permissions](https://docs.basistheory.com/api-reference/#permissions-permission-object) with “High Impact”. These two permissions allow your Application to create a new Token and read the value back when you need to access it.
```

We need to grant `token:general:read:high` permission based on the example reading back plaintext data. Otherwise, we have to override default privacy settings on the created token.